docker-compose.yml

```diff
@@ -1,7 +1,8 @@
 st2:
-  image: st2:latest
+  image: st2-upstart:latest
   container_name: st2
   hostname: st2
+  command: [$ST2_USER,$ST2_PASSWORD]
```

Comment:
Maybe we should bypass here and read these envvars directly inside `entrypoint.sh` because it's likely to happen in the near future that we'll add more and more options to `entrypoint.sh` for example list of services that you want to launch in this container, etc...

I'm doing that way
https://github.com/shusugmt/docker-st2/blob/master/entrypoint.sh